cp-deploy.sh

```diff
@@ -556,6 +568,14 @@ while (( "$#" )); do
     fi
     export CPD_SKIP_PORTABLE_REGISTRY=true
     shift 1
+    ;;
+  --clean-up)
+    if [[ "${ACTION}" != "apply" && "${ACTION}" != "destroy" && "${ACTION}" != "download"  ]];then
```

Comment:
ACTION == "vault" also qualifies for clean-up

Reply:
Thanks for the feedback. I have added a new commit to make the `--clean-up` cmd work with the `vault` subcommand. This also meant changing `vault` to produce logs in the same was the the `environment` commands to give consistent output.

Reply:
@fketelaars I've also pointed the PR to the `airgap-misc` branch as requested.